.github/workflows/nodeapp.yml

```diff
@@ -0,0 +1,43 @@
+name: Node.js Application
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Why not push to the driver branches?

Reply:
Do you know if we update master branch's nodeapp.yml to specify driver-v1.0.0 branch for push/pull, will it trigger on this PR?

Reply:
Because there typically won't be pushes to the driver branches since they're release branches. All changes are from a different branch via PR, making it redundant.

Master is there more as a sanity check.

Reply:
No. There needs to be separate PRs for nodeapp.yml.